src/docs/integration/linux/overview.md

```diff
@@ -0,0 +1,78 @@
+
+## Reference Designs
+
+Peridio maintains opinionated reference implementations for a variety of development kits and evaluation boards. These designs apply a series of opinions to help produce a secure, robust, fault tolerant platform for designing connected products. These opinions are not requirements to implement Peridio into your product, Peridio is an extensible, modular core that can be integrated into a variety of systems and platforms. The Peridio Reference Designs for Linux implement the following opinions on structuring the platform which will prescribe a development workflow and update strategy.
+
+### Runtime Structure
+
+#### Partition Structure
+
+The following partition structure is a loose guide for how the system is organizaed. The partition structure is completely customizable, the following structure opinions were chosen as a general base line for producing fault tolerant embedded systems.
+
+* U-Boot Environment
+* Boot Part A/B Updates
+* RootFS A/B Updates (Squashfs read-only)
+* Overlay FS Data Partition (read write)
+
+The U-Boot Environment is used as a persistent data store for peridiod to track variables such as a boot counter, a validation flag, which partition slot is active, the contents of each slot, etc. These variables can be used at early boot time to perform roll back logic, for example, the validation flag can be set after the program reaches the desired runtime state after an update. If has not been set, something likely went wrong and the hardware watchdog reset the board. Fpr example:
+
+```text
+peridio_revert=\
+    if test ${peridio_active} = "a"; then\
+        echo "Reverting to partition B";\
+        setenv peridio_active "b";\
+    else\
+        echo "Reverting to partition A";\
+        setenv peridio_active "a";\
+    fi
+
+peridio_init=\
+    if test ${peridio_booted} = 1; then\
+        if test ${peridio_validated} = 0; then\
+            run peridio_revert;\
+            setenv peridio_validated 1;\
+            saveenv;\
+        fi;\
+    fi;\
+    if test ${peridio_active} = "a"; then\
+        setenv bootpart 2;\
+        setenv rootdevpath ${a.peridio_rootfs_part_devpath};\
+        setenv datadevpath ${a.peridio_datafs_part_devpath};\
+    else\
+        setenv bootpart 3;\
+        setenv rootdevpath ${b.peridio_rootfs_part_devpath};\
+        setenv datadevpath ${b.peridio_datafs_part_devpath};\
+    fi;
+
+bootcmd=run peridio_init my_boot_command
+```
+
+### Peridio Cloud Requirements
+
+Building and connecting a reference design system to Peridio requires some additional configuration in your Peridio Cloud organization. The following steps wll prepare your Peridio Cloud organization to connect devices running the default reference platform.
+
+* Reference designs use the product name `avocado` by default, [Create a new product](/platform/guides/creating-products) named `avocado` in your Peridio Cloud organization.
```

Comment:
This section ("Peridio Cloud Requirements") seems to be talking about reference designs in general (not just Linux) which is ok, but we say:

> Reference designs use the product name avocado by default, [Create a new product](http://localhost:3000/platform/guides/creating-products) named avocado in your Peridio Cloud organization.

Which makes me think the default _product name_ should be unique across at least systems (linux, android, rtos) but probably even across build tools and OSes (linux buildroot, linux yocto, android, rtos zephyr, rtos FreeRTOS). The justification being imagine the customer who up front or over time wants to try out multiple reference designs - them all clashing on the same product namespace in Peridio would be a bad experience.

Reply:
updated to `avocado-linux`